source/_posts/2018-09-30-hacktoberfest.markdown

```diff
@@ -0,0 +1,56 @@
+
+## {% linkable_title Home Assistant %}
+
+- Get familiar with our [Style Guidelines](https://developers.home-assistant.io/docs/en/development_guidelines.html) and the existing code.
+- Go through the [Developer documentation](https://developers.home-assistant.io).
+- Keep your Pull requests small. One change per Pull request is faster to review than 19 changes. The same applies to new integrations. Start with the component and one platform. Add the other platforms later as this would get you feedback which you can incorporate in the coming platforms.
+- Don’t forget the documentation. New integrations need documentation, and changes often lead to updates to documentation as well. You know how it works, but the users don’t.
+- Architectural changes need to be discussed before a change is proposed. Also, keep in mind that those kinds of Pull Request may not be reviewed during October.
+- If you update a dependency, then you need to make sure that it will work. This means, provide the configuration sample and other details (used devices, firmware version, etc.).
+
+## {% linkable_title Home Assistant Documentation %}
+
+- Make yourself familiar with [Markdown](https://developers.home-assistant.io/docs/en/documentation_index.html).
+- Understand our [Standards and Guidelines](https://developers.home-assistant.io/docs/en/documentation_standards.html).
+- For small changes like typos and clarifications, use the GitHub editor functionality. 
+- For more significant changes, we highly recommended that you set up an environment to build the documentation locally before opening a Pull Request.
+- Keep an eye on the branches. `current` is for changes on the existing documentation, `next` for new features or integrations that will be available with the next release.
+
+We are accepting Pull Request for other repositories (like frontend, hassio, etc.) as well. There the same rules will apply but the handling could be different.
+
+Hacktoberfest is not only about Pull Requests for us, it’s also about the whole ecosystem. Home Assistant depends on a lot of Python modules and you can help Home Assistant indirectly if you make other code more robust, migrate it to asyncio, make it work with Python 3.7, etc. Talk to the owner or the maintainer of the code to make your contributions count for Hacktoberfest.
+
+It would also be a huge help if you review other Pull Requests. This means that there is a bigger chance that your Pull Request will be handled in a timely manner. We already have a backlog of Pull Requests. If you think that you are not enough qualified to do it, don’t worry, there are people around who can and will assist you. It’s nothing wrong to go for the low hanging fruits first and multiple eyes always see more than one pair.
```

Comment:
I think this section needs to move upwards OR deserve its own header.

Reply:
Separated and with title.